src/extension/app-schema/app-schema-oracle-test/pom.xml

```diff
@@ -157,6 +157,21 @@
             <version>2.0.0-1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+	     	<groupId>com.fasterxml.jackson.core</groupId>
+	     	<artifactId>jackson-core</artifactId>
+	     	<scope>test</scope>
+	   	</dependency>
+	   	<dependency>
+		    <groupId>com.fasterxml.jackson.core</groupId>
+		    <artifactId>jackson-annotations</artifactId>
+		    <scope>test</scope>
+	    </dependency>
```

Comment:
Looks something is strange here whit the formatting, could you double check @fernandor777?

Reply:
Holy tabs batman, fixing.

Reply:
:guitar: